app/views/apps/_fields.html.haml

```diff
@@ -37,6 +37,39 @@
       = f.text_field :email_at_notices, :value => app_decorate.email_at_notices
       .field-helpertext times an error occurs (comma separated).
 
+%fieldset
+  %legend= t('.notice_fingerprinter')
+  %p= t('.notice_fingerprinter_text')
+  .checkbox
+    = check_box 'other', 'use_site_fingerprinter', {:checked => app_decorate.use_site_fingerprinter}, SiteConfig::CONFIG_SOURCE_SITE, SiteConfig::CONFIG_SOURCE_APP
+    = f.label :use_site_fingerprinter, t('.use_site_fingerprinter')
+
+  %div.custom_notice_fingerprinter{:style => app_decorate.custom_notice_fingerprinter_style}
```

Comment:
This might represent a potential partial view that could be shared with the site-wide configuration area.